addons/misc/CfgVehicles.hpp

```diff
@@ -166,9 +166,266 @@ class CfgVehicles {
             };
         };
     };
-
+    class Items_base_F;
+    class Kat_armbandRC: Items_base_F {
+        scope = 2;
+        scopeCurator = 2;
+        scopeArsenal = 2;
+        author = "Miss Heda, vccv9040 (Swedish Forces Pack)";
+        editorPreview = QPATHTOF(ui\ArmbandWhiteCross.paa);
+        model = QPATHTOF(models\armband\Armband.p3d);
+        hiddenSelections[] = {"Armband"};
+        hiddenSelectionsTextures[] = {QPATHTOF(models\armband\ArmbandWC.paa)};
+    };
+    class Kat_armbandRCM: Kat_armbandRC {
+        author = "Miss Heda, vccv9040 (Swedish Forces Pack)";
+        editorPreview = QPATHTOF(ui\ArmbandWhiteCrossMedic.paa);
+        model = QPATHTOF(models\armband\Armband.p3d);
+        hiddenSelectionsTextures[] = {QPATHTOF(models\armband\ArmbandWCM.paa)};
+    };
+    class Kat_armbandRCD: Kat_armbandRC {
+        author = "Miss Heda, vccv9040 (Swedish Forces Pack)";
+        editorPreview = QPATHTOF(ui\ArmbandWhiteCrossDoctor.paa);
+        model = QPATHTOF(models\armband\Armband.p3d);
+        hiddenSelectionsTextures[] = {QPATHTOF(models\armband\ArmbandWCD.paa)};
+    };
+    class Kat_armbandKAT: Kat_armbandRC {
+        author = "Miss Heda, vccv9040 (Swedish Forces Pack)";
+        editorPreview = QPATHTOF(ui\ArmbandKat.paa);
+        model = QPATHTOF(models\armband\Armband.p3d);
+        hiddenSelectionsTextures[] = {QPATHTOF(models\armband\ArmbandKAT.paa)};
+    };
     class Man;
     class CAManBase: Man {
+        class ACE_SelfActions
+        {
+            class ACE_Equipment
+            {
+                class Armband {
+                    displayName = CSTRING(Armband_Sling);
+                    condition = QUOTE([ARR_2(_player, ""kat_armband_red_cross"")] call ACEFUNC(common,hasItem) || [ARR_2(_player, ""kat_armband_medic"")] call ACEFUNC(common,hasItem) || [ARR_2(_player, ""kat_armband_doctor"")] call ACEFUNC(common,hasItem) || [ARR_2(_player, ""kat_armband_kat"")] call ACEFUNC(common,hasItem) || !(_player getVariable [ARR_2(QQGVAR(isLeftArmFree), false)]) || !(_player getVariable [ARR_2(QQGVAR(isRightArmFree), false)]) || !(_player getVariable [ARR_2(QQGVAR(isLeftLegFree), false)]) || !(_player getVariable [ARR_2(QQGVAR(isRightLegFree), false)]));
```

Comment:
i would just create new FUNC for this for easier reading

Reply:
I'm seeing what I can do, gonna update the PR tomorrow or later this week.